CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ Previously, it was overridden by any `className` passed in via optional params.
   'lift', 'fade', or 'none'. These values are passed to the underlying onsenNavigator component.
   ([#1641](https://github.com/xh/hoist-react/pull/1641))
 * `AppOption` configs now accept an `omit` property for conditionally excluding options.
+* `TabContainerModel` now accepts a `track` property for tracking tab views in admin track log. The
+  default value is set to `false`.
```

Comment:
Need to move this up to the snapshot version (common issue when a PR stays open across releases).